recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 1baf3d6b049db7cc6cbd7506524cdca45c41e3bfffb24814245fc18f5a8626c2
 
 build:
-  number: 201
+  number: 1201
   skip: True  # [win]
   features:
     - blas_{{ variant }}
```

Comment:
The toolchain is failing b/c of this if I understood it correctly.
@isuruf what is the recommended way to depend on packages built with openblas now?